src/main/java/com/accolite/tfr/controller/TestController.java

```diff
@@ -0,0 +1,87 @@
+package com.accolite.tfr.controller;
+
+import com.accolite.tfr.dto.FeatureDto;
+import com.accolite.tfr.dto.OrganisationDto;
+import com.accolite.tfr.dto.ProjectDto;
+import com.accolite.tfr.model.Feature;
+import com.accolite.tfr.model.Organisation;
+import com.accolite.tfr.model.Project;
+import com.accolite.tfr.service.OrganisationServiceImp;
+import com.accolite.tfr.service.ProjectServiceImp;
+//import com.accolite.tfr.service.Service;
+import org.springframework.beans.BeanUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/tfr")
+public class TestController {
+
+    @GetMapping("/testController")
+    public String testCtrl()
+    {
+        return "Working Successfully";
+    }
+
+    @Autowired
+    private ProjectServiceImp projectService;
+    @Autowired
+    private OrganisationServiceImp organisationService;
+
```

Comment:
please remove the commented lines

Reply:
Changes done as per required. Please check it once.

Reply:
approved